src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-document-card/sw-order-document-card.html.twig

```diff
@@ -133,6 +135,7 @@
                 <sw-context-menu-item
                     :disabled="!acl.can('document.viewer')"
                     class="sw-order-document-card__context-button-download-pdf"
+                    data-analytics-id="sw-order-document-card.download"
```

Comment:
`download-document` or remove the `document` part for `open-document` and `send-document`